kusto/test/etoe/ingestion_status_e2e_test.go

```diff
@@ -0,0 +1,395 @@
+package etoe

```

Comment:
This is a general comment:

I'm going to let this file go through, however......

Really, in the future this file should be combined with etoe_test.  Those tests should become table driven tests, where one of the tests simply passes the ReportResultToTable() option and the tests code knows about what extra things to check.